wework/ray
# Add Tabs to React-Product

JIRA Ticket: [WEB-517](https://jira.weworkers.io/browse/WEB-517)

Arch Review Doc: https://docs.google.com/document/d/1K4UBh2eJN_Ey4EV4siBEmMxcoJqAMa6bm2NlCliw8N4/edit#

**Change Details**

- Adds a Ray-Product Tabs component that allows for easy creation of tabs in a web page.

This PR also contains A LOT of infrastructure additions to make sure that things will compile correctly. 

**Screenshots, gifs, references**

![2019-08-28 12 28 48](https://user-images.githubusercontent.com/6375353/63886528-8a576380-c98f-11e9-9f7d-92c43cffb39a.gif)

**Packages / Dependencies**

None

**How to test this feature**

- `yarn storybook`
- View the Tabs component and its stories
- Try out the a11y stuff!
   - focus a tab
   - press ArrowLeft, ArrowRight, Home and End to see what happens

---
I Reviewed the Requirements:
- [x] PR is labeled correctly (`ready for review`, `package: ...`, `app: ...`, etc)
